MathGame.VincentyArmstrong/MathGame.VincentyArmstrong/UserInterface.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace MathGame.VincentyArmstrong;
+
+public class UserInterface
+{
+    private int questions;
+    private bool hardMode;
+    private List<Game> games = new();
+    private bool quit;
+
+    public void Run()
+    {
+        while (!quit)
+        {
+            MainMenuDisplay();
+            MainMenuInput();
+        }
+    }
+
+    private static void MainMenuDisplay()
+    {
+        Console.Clear();
+        Console.WriteLine(
+        @"
+            ---------------------------------------------
+            What game would you like to play today? Choose from the options below:
+            A - Addition
+            S - Subtraction
+            M - Multiplication
+            D - Division
+            H - View History
+            Q - Quit the program
+            ---------------------------------------------");
+    }
+   
+    private void MainMenuInput()
+    {
+        switch (Console.ReadLine().Trim().ToLower())
+        {
+            case "a":
+                Setup(GameType.Addition);
+                break;
+            case "s":
+                Setup(GameType.Subtraction);
+                break;
+            case "m":
+                Setup(GameType.Multiplication);
+                break;
+            case "d":
+                Setup(GameType.Division);
+                break;
+            case "h":
+                ViewHistory();
+                break;
+            case "q":
+                Exit();
+                break;
+            default:
+                break;
+        }
+    }
+
+    private void Setup(GameType gameType)
+    {
+        Questions();
+        Difficulty();
+        Game game = new Game(gameType, questions, hardMode);
+        games.Add(game);
+    }
+
+    private void Questions()
```

Comment:
This method name is unclear. It does not provide enough information as to what is going on in the method.

Consider:
```suggestion
    private void GetNumberOfQuestionsFromUser()
```